pages/common/tldr.md

```diff
@@ -14,3 +14,15 @@
 - Get help for a git subcommand:
 
 `tldr {{git checkout}}`
+
+- Update the local cache and rebuild index:
+
+`tldr -u`
+
+- Clear the local cache:
```

Comment:
I'm not sure about adding these last 2 examples, as they are not in the [official client specification](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md).

Reply:
So I would remove it. Applying.

Reply:
Changes committed to `patch-5`.